qnm_filter/gw_data.py

```diff
@@ -697,12 +698,29 @@ def load_noise_curve(self, attr_name, filename, ifo=None):
             the file name to be read.
         ifo : string, optional
             name of interferometer, by default None
+        df : float
+            frequency spacing of interpolated noise curve
         """
+
         filereader = np.loadtxt(filename)
+        equal_spacing = len(set(np.diff(filereader[:, 0]))) == 1 # check if the index spacing is equal
+        if equal_spacing:
+            interp_freqs = filereader[:, 0]
+            interp_data = filereader[:, 1]
+        else:
+            print("The loaded file did not have uniform frequency spacing. It has been interpolated.")
+            if f_low is None: f_low = 0
+            if f_high is None: f_high = filereader[-1,0]
+            interp = interp1d(filereader[:, 0], filereader[:, 1], bounds_error = False, fill_value = 1e-20)
```

Comment:
Disabling the error could be dangerous. I wouldn't implicitly set a default value for extrapolation.